packages/plugins/analytics/src/index.ts

```diff
@@ -0,0 +1,69 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the MIT License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019-Present Datadog, Inc.
+
+import { doRequest } from '@dd/core/helpers';
+import type { GlobalContext, PluginOptions } from '@dd/core/types';
+
+import { INTAKE_HOST, INTAKE_PATH, PLUGIN_NAME } from './constants';
+
+export { PLUGIN_NAME } from './constants';
+
+export const getAnalyticsPlugins = (context: GlobalContext): PluginOptions[] => {
+    const log = context.getLogger(PLUGIN_NAME);
+
+    context.sendLog = async (message: string, ctx: any = {}) => {
+        // Only send logs in production.
+        if (context.env !== 'production') {
+            return;
+        }
+
+        try {
+            const bundler = {
+                name: context.bundler.name,
+                version: context.bundler.version,
+            };
+
+            await doRequest({
```

Comment:
Wondering, is this worth awaiting? I'm not sure whether we want to delay the build if the network is slow, are there any downsides when logging asynchronously?

Reply:
I guess it can be up to the user of `context.sendLog()` to decide to await or not.

So the function does await, but you don't have to when you call it.

If the build exits prematurely it may cancel the call too, in case it's not awaited.

Reply:
> I guess it can be up to the user of context.sendLog() to decide to await or not.

ah true 🤦